content/applications/finance/accounting/bank/setup/bank_accounts.rst

```diff
@@ -1,5 +1,5 @@
 =============
-Bank Accounts
+Bank accounts
 =============
```

Comment:
I reckon you changed this because the title's capitalization was incorrect. But this change would need to be applied to older versions as well, and not only from 15.0.

So, I wouldn't put this change in this PR.